src/Http/Http.Extensions/src/TagsAttribute.cs

```diff
@@ -29,4 +31,10 @@ public TagsAttribute(params string[] tags)
     /// Gets the collection of tags associated with the endpoint.
     /// </summary>
     public IReadOnlyList<string> Tags { get; }
+
+    /// <inheritdoc/>>
+    public override string ToString()
+    {
+        return $"Tags: {string.Join(",", Tags)}";
```

Comment:
Would it be easier to read with `", "`?

Reply:
Comma + space indicates a new property. I didn't add spaces between items to make it more obvious they're in the same collection.